capsule-util/src/test/java/co/paralleluniverse/capsule/CapsuleLauncherTest.java

```diff
@@ -50,7 +52,7 @@ public void testSimpleExtract() throws Exception {
                 .addEntry("lib/a.jar", emptyInputStream())
                 .addEntry("lib/b.class", emptyInputStream())
                 .addEntry("q/w/x.txt", emptyInputStream())
-                .addEntry("d\\f\\y.txt", emptyInputStream()) // test with Windows path
+                .addEntry("d/f/y.txt", emptyInputStream())
```

Comment:
windows support not needed anymore? (minor?)

Reply:
Jar file separator (for its content) is always '/', it is not platform dependant